tests/v2/validation/snapshot/snapshot.go

```diff
@@ -35,21 +37,21 @@ const (
 	defaultNamespace             = "default"
 	DeploymentSteveType          = "apps.deployment"
 	isCattleLabeled              = true
-	IngressSteveType             = "networking.k8s.io.ingress"
+	initialIngress               = "ingress-before-restore"
+	initialWorkload              = "wload-before-restore"
+	ingressSteveType             = "networking.k8s.io.ingress"
 	ingressPath                  = "/index.html"
-	initialIngressName           = "ingress-before-restore"
-	initialWorkloadName          = "wload-before-restore"
 	localClusterName             = "local"
 	K3S                          = "k3s"
 	kubernetesVersion            = "kubernetesVersion"
 	namespace                    = "fleet-default"
 	port                         = "port"
+	postWorkload                 = "wload-after-backup"
 	ProvisioningSteveResouceType = "provisioning.cattle.io.cluster"
 	RKE1                         = "rke1"
 	RKE2                         = "rke2"
 	serviceAppendName            = "service-"
-	ServiceType                  = "service"
-	WorkloadNamePostBackup       = "wload-after-backup"
```

Comment:
since some of these were public, can you confirm you double checked they aren't used elsewhere via import?

Reply:
These were public as an accident, I just fixed them as they are only used here